dashboard/config/environments/test.rb

```diff
@@ -35,6 +37,11 @@
     config.i18n.backend = Cdo::I18n::LazyLoadableBackend.new(lazy_load: true)
   end
 
+  # In CI environments (ie, Drone), stub AWS SageMaker so we can run UI tests
+  if is_ci
+    config.stub_sagemaker = true
```

Comment:
Should this be a global `stub_external_aws_services` (or a better name) so we can read it in other helpers too (ex. comprehend)?

Reply:
hmm we have some? (one? a lot of?) AWS services that we do not stub in Drone (eg, S3), so I think that broad of a name might be confusing...that said, this flag doesn't really globally stub SageMaker (you still have to use the flag in the appropriate place), so if we want Comprehend to share the same flag, we could do something like `stub_aichat_aws_services` and comment which services those are?

Reply:
Updated to that, LMK what ya think!

Reply:
Yep that works! Thinking that we might eventually be able to re-enable Javalab tests with this too, but we can figure out a different naming solution if/when we get there